src/6_branch.js

```diff
@@ -459,9 +459,10 @@ Branch.prototype['init'] = wrap(
 		};
 
 		if (sessionData && sessionData['session_id'] && !link_identifier) {
+			// resets data in session storage to prevent previous link click data from being returned to Branch.init()
+			session.update(self._storage, { "data": "" });
```

Comment:
@rubinsingh do we want to set data to and empty string? or would it make more sense to set it to null?